Dockerfile

```diff
@@ -16,19 +16,27 @@ COPY controllers/ controllers/
 COPY utils/ utils/
 COPY templates/ templates/
 COPY readinessProbe/ readinessProbe/
+COPY cmd/awsDataGather/ awsDataGather/
+COPY pkg/ pkg/
+
+RUN mkdir bin
```

Comment:
I put the main program for aws data gathering in `cmd/` and its auxiliary objects (which is also used by the deployer) in `pkg/` to try out a [common layout pattern for go projects](https://github.com/golang-standards/project-layout). I'm interested to know others thoughts on this. If it looks promising, I can move the readinessProbe/, utils/ and templates/ as well.